low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -410,77 +436,92 @@ def _generate_host_update(self):
                 self._host_memory_usage_threshold = float(self._host_memory_usage_threshold)
         except ValueError:
             logger.warning("Host Memory Alert, Invalid host_memory_usage_threshold value are entered in config.")
-            # Assigning default value to _disk_usage_threshold
+            # Assigning default value to _memory_usage_threshold
             self._host_memory_usage_threshold = self.DEFAULT_HOST_MEMORY_USAGE_THRESHOLD
 
+        memory_persistent_data = self.read_persistent_data('MEMORY_USAGE_DATA')
+        if memory_persistent_data.get('memory_usage_time_map') is not None:
+            previous_check_time = int(memory_persistent_data['memory_usage_time_map'])
+        else:
+            previous_check_time = int(-1)
+        self.usage_time_map['memory'] = current_time

```

Comment:
you can directly use time.time() here. One liner functions are not generally useful (`get_current_time`)

Reply:
Same as previous comment, it is one liner function, but it has been used multiple places so would be easier to make changes in future if it is a function.

Reply:
agree